grakn-engine/src/test/java/ai/grakn/engine/config/GraknEngineConfigTest.java

```diff
@@ -35,6 +39,9 @@
 
     private GraknEngineConfig configuration = GraknEngineConfig.create();
 
+    @ClassRule
+    public static MockRedisRule mockRedisRule = MockRedisRule.create(new SimpleURI(EngineTestHelper.config().getProperty(GraknEngineConfig.REDIS_HOST)).getPort());
```

Comment:
Maybe we should have a helper method to do this, since we do the same thing several times.

Also, I don't understand why this test needs `MockRedis`, but then `GraknEngineConfig` is a scary place.

Reply:
I tried to make a helper method. The problem is that any helper method will have go into `MockRedisRule` which is in `grakn-test-tools`. This helper method will need to use the config `GraknEngineConfig` which is in `grakn-engine`. 

So that means making the helper method re-read the config (ewww) and that is a bad idea because often tests override options in the config. This is the least evil approach for the moment.

Reply:
OK, I see! This seems reasonable then.